tmux/hl2ss_demo.yml

```diff
@@ -0,0 +1,79 @@
+#
+# System configuration to run the ANGEL system for the 2022/11 PI meeting and
+# Evaluation 1.
+#
+# This configuration is for the Coffee task/recipe.
+#
+
+name: HL2SS Video Streamer
+root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
+
+# Optional tmux socket
+# socket_name: foo
+
+# Note that the pre and post options have been deprecated and will be replaced by
+# project hooks.
+
+# Project hooks
+
+# Runs on project start, always
+# on_project_start: command
+on_project_start: |
+  # Current DemoUI component assumes "/debug" namespace.
+  #export ROS_NAMESPACE=${ROS_NAMESPACE:-/debug}
+  export ROS_NAMESPACE=/debug
+  export HL2_IP=${HL2_IP:-192.168.86.46}
+  export CONFIG_DIR=${ANGEL_WORKSPACE_DIR}/config
+  export NODE_CONFIG_DIR=${ANGEL_WORKSPACE_DIR}/src/angel_system_nodes/configs
+  export MODEL_DIR=${ANGEL_WORKSPACE_DIR}/model_files
+
+# Run on project start, the first time
+# on_project_first_start: command
+
+# Run on project start, after the first time
+# on_project_restart: command
+
+# Run on project exit ( detaching from tmux session )
+# on_project_exit: command
+
+# Run on project stop
+# on_project_stop: command
+
+# Runs in each window and pane before window/pane specific commands. Useful for setting up interpreter versions.
+# pre_window: rbenv shell 2.0.0-p247
+
+# Pass command line options to tmux. Useful for specifying a different tmux.conf.
+# tmux_options: -f ~/.tmux.mac.conf
+tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
+
+windows:
+  - hl2ss: ros2 run angel_system_nodes hl2ss_ros_bridge --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p ip_addr:=${HL2_IP}
+      -p image_topic:=PVFramesRGB
+      -p hand_pose_topic:=HandJointPoseData
+
+  # Visualize RGB Images being output from the headset
+  - rqt_2d_detections: rqt -s rqt_image_view/ImageView
+      --args ${ROS_NAMESPACE}/PVFramesRGB
+      --ros-args -p _image_transport:=raw
+  # Object detector
+  - object_detector: ros2 run angel_system_nodes object_detector_with_descriptors --ros-args
```

Comment:
Running the object detector is overkill for demonstrating data stream output from the HL2SS interaction. If we had something to show/render the hand pose output, that would be better.